drizzlepac/alignimages.py

```diff
@@ -864,7 +837,7 @@ def determine_fit_quality(imglist, filtered_table, catalogs_remaining, print_fit
     overall_valid = True
     overall_comp = False
     for item in imglist:
-        if not item.meta['fit_info']['status'].startswith('FAILED'):
+        if item.meta['fit_info']['status'].startswith('FAILED') != True:
```

Comment:
"!=" operator should not be used with Booleans. Maybe replace "!= True" with "is False"

Reply:
Will replace with 'is False'.

Reply:
Resolved in 51e3cea